src/main/webapp/webpack/i18nThymeleafWebpackPlugin.js

```diff
@@ -0,0 +1,141 @@
+"use strict";
+
+/**
+ * @param {string[]} keys the translation keys required for the entry
```

Comment:
Again a great big comment at the top of this file about what this is doing please.

Reply:
Let me leave this until the next one since this gets a big refactor.

Reply:
Updated in 12c774be5c043d69233d86c55bf8a1ce3b8c8a99